portal/server/controllers/dashboard/deals/index.test.js

```diff
@@ -69,6 +69,7 @@ describe('controllers/dashboard/deals', () => {
       params: { page: 1 },
       session: {
         dashboardFilters: CONSTANTS.DASHBOARD.DEFAULT_FILTERS,
+        sortBy: CONSTANTS.DASHBOARD.DEFAULT_SORT.order,
```

Comment:
Need a test for a user journey where session does not exists please.

Reply:
done